### ISSUE TITLE
[Question] Can i run parallel test using WS ?

### ISSUE THREAD

**Issue by amrsa1** [microsoft/playwright#13461]
is it possible to connect to ws and execute the test in parallel against different browsers ?

**Comment by mxschmitt** [comment]
Do you mean something like that? https://github.com/microsoft/playwright/issues/8379

Or do you mean that you want to have a single browser.launchServer somewhere and all the worker connect to them via BrowserType.connect?

**Comment by amrsa1** [comment]
@mxschmitt not really, but I meant to be able to connect to multiple browsers using ws instead of defining a particular browser type.

so let's say to execute the same tests against three different browsers on the same instance and using wsEndpoint , instead of explicitly defining one browser type 

`wss://aerokube-moon.somewhere-locally/playwright/chromium/playwright-1.19.2`

**Comment by mxschmitt** [comment]
gotcha, yeah then its a duplicate of https://github.com/microsoft/playwright/issues/8379

Folding into that.
